aips/aip-hybrid-nft-standard.md

```diff
@@ -0,0 +1,352 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Aptos Hybrid Asset (HA) Standard
+author: @gregnazario, @cylim226, @decidethatlater, @HugeMongooose, @SuperSecretRare, @0xYZMIN
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): 10/15/2024
+type: Ecosystem
+created: 10/3/2024
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Aptos Hybrid Asset (HA) Standard
+
+## Summary
+
+This AIP proposes a standardized implementation for hybrid assets on the Aptos blockchain, inspired by Vectorized's [DN-404](https://github.com/Vectorized/dn404) token standards. 
```

Comment:
Is this going to be part of the framework or just a standard enforced through common interface/interactions?